extensions/amp-selector/0.1/test/test-amp-selector.js

```diff
@@ -1164,6 +1164,38 @@ describes.realWin(
           }
         );
 
+        it(
+          'should update focus when the user presses the home/end keys when ' +
+            'keyboard-select-mode is enabled',
+          () => {
+            const ampSelector = getSelector({
+              attributes: {
+                'keyboard-select-mode': 'focus',
+              },
+              config: {
+                count: 3,
+              },
+            });
+            ampSelector.children[1].setAttribute('selected', '');
+            ampSelector.build();
+            expect(ampSelector.children[0].tabIndex).to.equal(-1);
+            expect(ampSelector.children[1].tabIndex).to.equal(0);
+            expect(ampSelector.children[2].tabIndex).to.equal(-1);
+            return keyPress(ampSelector, Keys.HOME)
```

Comment:
Should add another test making sure the disabled item skipping works for both home/end.